src/store.js

```diff
@@ -0,0 +1,26 @@
+import Vue from 'vue';
+import Vuex from 'vuex';
+
+Vue.use(Vuex);
+
+export default new Vuex.Store({
+  state: {
+    gender: '男性',
+    year: '',
+    month: '',
+    day: '',
+    insurance: '',
+    condition: '',
+    history: '',
+    question:''
+  },
+  mutations: {
+    displayGender(gen) {
+      if(gen === 1){
```

Comment:
displayGenderは今後使用する予定のmutationsでしょうか？
であればいいのですが、もし不要であれば削除しましょう。

また、`()`の両外側にはスペースをあけましょう。エディターの自動整形を利用するといいですね。
```suggestion
      if (gen === 1) {
```

Reply:
課題2-2の実装を考えていた際に残してしまっておりました。

Prittierをインストールしまして、自動整形してみました！
ご指摘ありがとうございます！